src/main/java/com/minecolonies/coremod/network/messages/TransferRecipeCrafingTeachingMessage.java

```diff
@@ -0,0 +1,66 @@
+package com.minecolonies.coremod.network.messages;
+
+import com.minecolonies.api.util.ItemStackUtils;
+import com.minecolonies.coremod.inventory.CraftingGUIBuilding;
+import io.netty.buffer.ByteBuf;
+import net.minecraft.entity.player.EntityPlayerMP;
+import net.minecraft.item.ItemStack;
+import net.minecraftforge.fml.common.network.ByteBufUtils;
+import net.minecraftforge.fml.common.network.simpleimpl.IMessage;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class TransferRecipeCrafingTeachingMessage extends AbstractMessage<TransferRecipeCrafingTeachingMessage, IMessage>
+{
+    Map<Integer, ItemStack> itemStacks = new HashMap<>();
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Explicitly declare the visibility for "itemStacks". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS2039)